src/Logary/Registry.fs

```diff
@@ -206,6 +214,59 @@ module Registry =
          && not ("true" = Env.varDefault "LOGARY_I_PROMISE_I_HAVE_PURCHASED_LICENSE" (fun () -> "false")) then
          failwith "You must purchase a license for Logary to run it on or with IIS or Kestrel."
 
+    // check if msg has enable metric counter
+    let hookupMetric (metricRegistry: MetricRegistry) (message: Message) =
```

Comment:
This is what I was thinking could be a default `Flow` middleware.

Reply:
Yes, make it as a middleware.